doc/auto_examples/workflows/plot_simulate_evoked.rst

```diff
@@ -0,0 +1,466 @@
+
+    net_sync.add_evoked_drive(
+        'evprox2', mu=137.12, sigma=8.33, numspikes=1,
+        weights_ampa=weights_ampa_p2, location='proximal', n_drive_cells=n_drive_cells,
+        cell_specific=cell_specific, synaptic_delays=synaptic_delays_prox, event_seed=4)
+
+
+
+
+
+
+
+
+.. GENERATED FROM PYTHON SOURCE LINES 142-143
+
+You may interrogate current values defining the spike event time dynamics by
+
+.. GENERATED FROM PYTHON SOURCE LINES 143-145
+
+.. code-block:: default
+
+    print(net_sync.external_drives['evdist1']['dynamics'])
+
+
+
+
+
+.. rst-class:: sphx-glr-script-out
+
+ Out:
+
+ .. code-block:: none
+
+    {'mu': 63.53, 'sigma': 3.85, 'numspikes': 1}
+
+
+
+
+.. GENERATED FROM PYTHON SOURCE LINES 146-148
+
+Finally, let's simulate this network. Rather than modifying the dipole
+object, this time we make a copy of it before smoothing and scaling.
+
+.. GENERATED FROM PYTHON SOURCE LINES 148-154
+
+.. code-block:: default
+
+    dpls_sync = simulate_dipole(net_sync, tstop=170., n_trials=1)
+
+    trial_idx = 0
+    dpls_sync[trial_idx].copy().smooth(window_len).scale(scaling_factor).plot()
+    net_sync.cell_response.plot_spikes_hist()
+
+
+
+
+.. rst-class:: sphx-glr-horizontal
+
+
+    *
+
+      .. image-sg:: /auto_examples/workflows/images/sphx_glr_plot_simulate_evoked_004.png
+         :alt: Aggregate (L2 + L5)
+         :srcset: /auto_examples/workflows/images/sphx_glr_plot_simulate_evoked_004.png
+         :class: sphx-glr-multi-img
+
+    *
+
+      .. image-sg:: /auto_examples/workflows/images/sphx_glr_plot_simulate_evoked_005.png
+         :alt: plot simulate evoked
+         :srcset: /auto_examples/workflows/images/sphx_glr_plot_simulate_evoked_005.png
+         :class: sphx-glr-multi-img
+
+
+.. rst-class:: sphx-glr-script-out
+
+ Out:
+
+ .. code-block:: none
+
+    joblib will run over 1 jobs
+    Loading custom mechanism files from /storage/home/m/mzk5905/hnn-core/hnn_core/mod/x86_64/.libs/libnrnmech.so
+    Building the NEURON model
+    [Done]
+    running trial 1 on 1 cores
+    Simulation time: 0.03 ms...
+    Simulation time: 10.0 ms...
+    Simulation time: 20.0 ms...
+    Simulation time: 30.0 ms...
+    Simulation time: 40.0 ms...
+    Simulation time: 50.0 ms...
+    Simulation time: 60.0 ms...
+    Simulation time: 70.0 ms...
+    Simulation time: 80.0 ms...
+    Simulation time: 90.0 ms...
+    Simulation time: 100.0 ms...
+    Simulation time: 110.0 ms...
+    Simulation time: 120.0 ms...
+    Simulation time: 130.0 ms...
+    Simulation time: 140.0 ms...
+    Simulation time: 150.0 ms...
+    Simulation time: 160.0 ms...
+
+    <Figure size 640x480 with 1 Axes>
+
+
+
+.. GENERATED FROM PYTHON SOURCE LINES 155-160
+.. warning::
+   Always look at dipoles in conjunction with raster plots and spike histogram to avoid misinterpretation.
+
+   Run multiple trials for your simulation to get an average of different drives seeds before drawing conclusions.
```

Comment:
You should put this in the python script as comments. This file you edited is auto generated ...

Reply:
> You should put this in the python script as comments. This file you edited is auto generated ...

@jasmainak  👍  I thought adjusting the .rst file is how you do it :)
will update the python script then make a new pull request.